src/daemon/ca/certauth.rs

```diff
@@ -224,6 +224,9 @@ impl Aggregate for CertAuth {
                         }
                     }
                 }
+                for suspended in suspended {
```

Comment:
"for suspended in suspended" looks confusing, would this work and be better as "suspended.for_each()" instead?